<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58,58,58);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58,58,58);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58,58,58);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="UserFrame">
         <property name="geometry">
          <rect>
           <x>41</x>
           <y>264</y>
           <width>300</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(131, 131, 131);
	
}</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Usu√°rio</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_5">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>200</width>
           <height>200</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/home/Icons/LOGOPROEXPRESS2-removebg-preview.png);
background-color: rgba(255, 255, 255, 0.4);
border: 5px solid rgb(45,45,45);
border-radius: 15px;
padding: 15px;
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="PasswordFrame">
         <property name="geometry">
          <rect>
           <x>41</x>
           <y>329</y>
           <width>300</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(131, 131, 131);
	
}</string>
         </property>
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Senha</string>
         </property>
        </widget>
        <widget class="QPushButton" name="EnterButton">
         <property name="geometry">
          <rect>
           <x>51</x>
           <y>411</y>
           <width>281</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(93, 93, 93);
border: 2px solid rgb(132, 132, 132);
border-radius: 5px
}

QPushButton:hover {
background-color: rgb(73, 73, 73);
border: 2px solid rgb(102, 102, 102);
border-radius: 5px
}</string>
         </property>
         <property name="text">
          <string>Entrar</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58,58,58);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
